client/capstone-app/src/components/RouteCard.jsx

```diff
@@ -96,12 +96,16 @@ const RouteCard =  ({tripIds, tripHeadsign, routeId, startStopId, endStopId, sta
        (<div>
             <div>{tripHeadsign} --- {transfers[0].tripHeadsign}</div>
             <div>{routeId} --- {transfers[0].routeId}</div>
+            {data.isDelayed === true || transfers[0].isDelayed === true &&
+            <p style={{color: "red"}}>Late!</p>}
             <Button onPress={onOpen} isOpen={isOpen} onOpenChange={onOpenChange}>See Trip Details</Button>
             <RouteInfo directions={tripDetails} isOpen={isOpen} onOpenChange={onOpenChange} data = {data} currentCoordinates={currentCoordinates} destinationCoordinates={destinationCoordinates}/>
        </div>):
        (<div>
             <div>{tripHeadsign}</div>
             <div>{routeId}</div>
+            {data.isDelayed === true &&
```

Comment:
You can do `data.isDelayed` and not `data.isDelayed===true` assuming that you defined isDelayed to be a boolean type. For boolean vars, you do not need to do the evaluation since x === true is equivalent to x=true

Reply:
Thanks for the catch! Will fix it in the next commit!